oras/auth/base.py

```diff
@@ -17,6 +19,9 @@ class AuthBackend:
     Generic (and default) auth backend.
     """
 
+    session: requests.Session
+    _tls_verify: bool
```

Comment:
Why would you make the tls verify a class variable? This means multiple classes must have the same state, and if you change one the others follow.

Reply:
The base class [does have an init](https://github.com/oras-project/oras-py/blob/938390ae8d283ded95924c9fc87ffeea53d38e2d/oras/auth/base.py#L20) - it primarily doesn't include many variables that were added after.